cedar-policy-core/src/evaluator/err.rs

```diff
@@ -42,19 +41,20 @@ pub enum EvaluationError {
     #[error("cannot access attribute of unspecified entity: {0}")]
     UnspecifiedEntityAccess(SmolStr),
 
-    /// Tried to get this attribute of a (non-entity) record, but that record
+    /// Tried to get an attribute of a (non-entity) record, but that record
     /// didn't have that attribute
-    #[error("record does not have the required attribute: {0}")]
+    #[error("record does not have the attribute: {0}")]
```

Comment:
Do we need "\`{0}\`"?

Reply:
Our conventions state that:

* Formatted strings that display user-provided inputs should go after a colon : or, if that is not possible, be surrounded by backticks ``.
    * E.g., "duplicate template or policy id: {0}"
    * E.g.  "error occurred while evaluating policy `{0}`: {}"